include/seqan3/core/platform.hpp

```diff
@@ -127,6 +127,29 @@
      */
 #endif
 
+// Lemon [optional]
+/*!\def SEQAN3_WITH_LEMON
+ * \brief Whether Lemon support is available or not.
+ * \ingroup core
+ */
+#ifndef SEQAN3_WITH_LEMON
+#   if __has_include(<lemon/config.h>)
+#       define SEQAN3_WITH_LEMON 1
+#   else
+#       define SEQAN3_WITH_LEMON 0
+#   endif
+#elif SEQAN3_WITH_LEMON != 0
+#   if !__has_include(<lemon/config.h>)
+#       error Lemon was marked as required, but not found!
+#   else
+#       define LEMON_HAVE_LONG_LONG 1
+#       define LEMON_USE_PTHREAD 1
+#       define LEMON_CXX11 1
+#       define LEMON_USE_WIN32_THREADS 1
+#       define LEMON_WIN32 0
```

Comment:
For the above two: I assume "win32" here just means windows, in that case "#ifndef __unix__` (or make an if- else with above).

But, please double check, that they mean "windows" and no "32bit windows"

Reply:
Yes, it means Windows in general.